oeplatform/securitysettings.py.default

```diff
@@ -16,12 +16,12 @@ token_test_user = '0'
 # https://docs.djangoproject.com/en/1.8/ref/settings/#databases
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.postgresql_psycopg2',
-        'NAME': 'oep_django',
-	'USER': os.environ.get("OEP_DB_USER"),
-	'PASSWORD': os.environ.get("OEP_DB_PASSWORD"),
-	'HOST': 'localhost'                      
-	}
+    'ENGINE': 'django.db.backends.postgresql_psycopg2',
+    'NAME': 'oep_django',
+    'USER': os.environ.get("OEP_DB_USER"),
```

Comment:
"OEP_DB_USER" could be changed to "oep_django_user" as is the placeholder in the README.md . Similarly "OEP_DB_PASSWORD" could be changed to "oep_django_password". Do you agree @Bachibouzouk ? This was mildly distracting during the tutorial, but nothing major.

Reply:
Never mind. I see that this was addressed in the README right after.